projects/observability/src/shared/components/cartesian/cartesian-chart.component.ts

```diff
@@ -16,19 +16,22 @@ import { LegendPosition } from '../legend/legend.component';
 import { ChartTooltipBuilderService } from '../utils/chart-tooltip/chart-tooltip-builder.service';
 import { DefaultChartTooltipRenderData } from '../utils/chart-tooltip/default/default-chart-tooltip.component';
 import { MouseLocationData } from '../utils/mouse-tracking/mouse-tracking';
-import { Axis, AxisLocation, AxisType, CartesianChart, RenderingStrategy, Series } from './chart';
+import { Axis, AxisLocation, AxisType, CartesianChart, Range, RenderingStrategy, Series } from './chart';
 import { ChartBuilderService } from './chart-builder.service';
 import { defaultXDataAccessor, defaultYDataAccessor } from './d3/scale/default-data-accessors';
 
 @Component({
   selector: 'ht-cartesian-chart',
-  changeDetection: ChangeDetectionStrategy.OnPush,
   styleUrls: ['./cartesian-chart.component.scss'],
-  template: ` <div #chartContainer class="fill-container" (htLayoutChange)="this.redraw()"></div> `
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `<div #chartContainer class="fill-container" (htLayoutChange)="this.redraw()"></div> `
 })
 export class CartesianChartComponent<TData> implements OnChanges, OnDestroy {
   @Input()
-  public series: Series<TData>[] | undefined = [];
+  public series?: Series<TData>[] | undefined = [];
+
+  @Input()
+  public range?: Range<TData>;
```

Comment:
we don't support multiple ranges? Also, I'd recommend changing this name, since range already has a meaning in the context of a cartesian, and shows up in a number of places in code.

Reply:
I'm quite bad at naming too - here's a bit of brainstorming:
`Band`
`SeriesPair`
`SeriesBand`
`SeriesBounds`

Reply:
I almost went with Band. It just didn't sound as cool. Will change to that.

Reply:
Band doesn't sound cool?! Bands are the coolest!